src/Raven.Server/Documents/Queries/IndexQueryServerSide.cs

```diff
@@ -35,6 +36,9 @@ public class IndexQueryServerSide : IndexQuery<BlittableJsonReaderObject>
         [JsonDeserializationIgnore]
         public SpatialDistanceFieldComparatorSource.SpatialDistanceFieldComparator Distances;
 
+        [JsonDeserializationIgnore]
+        public CancellationToken Token;
```

Comment:
please remove this, we are passing token as a separate parameter everywhere else

Reply:
done